sources/packages/api/src-sql/MSFAANumbers/Create-msfaa-numbers-table.sql

```diff
@@ -0,0 +1,39 @@
+CREATE TABLE IF NOT EXISTS sims.msfaa_numbers (
+  id SERIAL PRIMARY KEY,
+  msfaa_number BIGINT NOT NULL,
+  date_requested TIMESTAMP WITH TIME ZONE,
+  date_signed DATE,
+  service_provider_received_date DATE,
+  -- Reference Columns
+  student_id INT NOT NULL REFERENCES sims.students(id) ON DELETE CASCADE,
+  -- Audit columns
+  created_at timestamp without time zone NOT NULL DEFAULT now(),
+  updated_at timestamp without time zone NOT NULL DEFAULT now(),
+  creator INT NULL DEFAULT NULL REFERENCES sims.users(id) ON DELETE
+  SET
+    NULL,
+    modifier INT NULL DEFAULT NULL REFERENCES sims.users(id) ON DELETE
```

Comment:
please remove the extra space

Reply:
The "extra space" is due to a SQL Formatter (https://marketplace.visualstudio.com/items?itemName=adpyke.vscode-sql-formatter).
The idea was to try to have something common between us in the same way that we have for Typescript.
It could not be what we have been doing but maybe we can all use the same VS Code extension (the one that I mentioned or any other) to keep the same formatting.
@ann-aot @dheepak-aot, thoughts?

Reply:
mm, I don't think it's an extra white space, I believe its part of formatting, and I believe we are following this format in our  `.sql` file, for eg, 
![image](https://user-images.githubusercontent.com/77353155/132918960-c2be039d-00b7-4ccf-bfa2-71cb27766171.png)